modules/etc/commands/query.go

```diff
@@ -0,0 +1,44 @@
+package commands
+
+import (
+	"encoding/hex"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	cmn "github.com/tendermint/tmlibs/common"
+
+	"github.com/tendermint/basecoin/client/commands"
+	"github.com/tendermint/basecoin/client/commands/query"
+	"github.com/tendermint/basecoin/modules/etc"
+	"github.com/tendermint/basecoin/stack"
+)
+
+// EtcQueryCmd - command to query raw data
+var EtcQueryCmd = &cobra.Command{
+	Use:   "etc [key]",
+	Short: "Get data stored under key in etc",
+	RunE:  commands.RequireInit(etcQueryCmd),
+}
```

Comment:
what's etc again? Maybe we should add more documentation - `etc` not very descriptive. If this command is supposed to just be querying raw data from the `eyes` (in general, as this commands code-comment suggests) then `etc` is maybe a bad descriptor of the functionality - however `Short:` desc seems to suggest otherwise